bun.lock

```diff
@@ -23,7 +23,7 @@
         "@react-router/node": "7.6.2",
         "@scure/bip32": "1.6.2",
         "@scure/bip39": "1.5.4",
-        "@supabase/supabase-js": "2.50.0",
+        "@supabase/supabase-js": "2.49.8",
```

Comment:
this update actually broke something with realtime and reconnection is not working with that or latest version (2.50.2) so I had to revert it. I lost so much time to figure it out 🤦